src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,49 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ title, description, imgUrl, imdbUrl }) => (
```

Comment:
```suggestion
export const MovieCard = ({
  title,
  description,
  imgUrl,
  imdbUrl
}) => (
```
I think It is better

Reply:
If the line is less than 70 chars and easy to read it is not necessary.